mautic/mautic
# [UX] Improve labels system

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [❌]
| New feature/enhancement? (use the a.x branch)      | [✅]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [❌]
| Automated tests included?              | [❌] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13158 



#### Description:
This PR adds a set of new tags/label styles for devs to use while building new features.
It also changes some labels to avoid primary, warning and success being used where they are inadequate.
These tags have hover effect to improve user experience.
Aiming to provide consistency, I also added an operational tag to be triggered when `<a>` is being used. This way, clickable tags will have a darker border color to indicate for users that they're interactive.

![image](https://github.com/mautic/mautic/assets/116097999/e9db4f0c-f5d1-436f-9c2b-4cabe26f9b6a)

(company score was just the number, no styles before)
![image](https://github.com/mautic/mautic/assets/116097999/bcc81290-c9c2-4a77-b5be-effc96911de3)

![image](https://github.com/mautic/mautic/assets/116097999/3c268636-a332-4d53-b3e9-b79f9e285d04)

chosen (library) now follow a standard, which indicates these labels are selectable/in the context of filtering options
![image](https://github.com/mautic/mautic/assets/116097999/4e8a1bd7-5ebc-45dc-bed8-51fedf8d755e)






#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Companies > Create > Back to the list > See
3. Go to Emails > Create one, save and close > See how it appears on the table
4. Go to Contacts > Create > Save & Close > Scroll down to Filters > Add one
5. Go to Categories > Create one with color > Back to list and see
6. Go to Assets > Create one > Assign the colored category > Back to the list and see
7. Go to Campaigns > Create one > Assign the colored category > Back to the list and see
8. Go to Dynamic Content > Create one > Assign the colored category > Back to the list and see
9. Go to Forms > Create one > Back to the list and see the Submissions count
10. Go to Contacts > Import > Select a file > Proceed with the import > Select to import in the background > Then go to the imports list to see the labels
11. Go to Points > Triggers > Create new > Back to the list to see its color
